### PR TITLE
Fix bug/tax declaration

### PR DESCRIPTION
**Pull request by Sanchodelniglo**
First commit is item 766
others are item 763
sorry for that !

### CODE REVIEW

File: app/views/backend/bank_reconciliation/items/_reconciliation.html.haml [hunk 1]
```diff
@@ -22,10 +22,10 @@
       - url_redirect_params[:period_start] = params[:period_start] if params[:period_start].present?
       - url_redirect_params[:period_end] = params[:period_end] if params[:period_end].present?
 
-      -# = dropdown_menu_button :new_purchase, icon: false do |menu|
```
Comment (piotaixr) [inline app/views/backend/bank_reconciliation/items/_reconciliation.html.haml]: Remove the comment?
Same below :)

File: app/views/backend/tax_declarations/index.html.haml [hunk 2]
```diff
@@ -11,4 +11,5 @@
       - export_params = params.slice(:period, :state)
       - d.item :export_as_pdf.tl, params: export_params.merge(format: :pdf)
       - d.item :export_as_csv.tl, params: export_params.merge(format: :csv)
+
```
Comment (piotaixr) [inline app/views/backend/tax_declarations/index.html.haml]: Can you remove the blank line?

File: app/views/layouts/_side.html.haml [hunk 3]
```diff
@@ -8,6 +8,7 @@
           - attrs[:active] = true if item == current[:item]
           - m.link(item, item.default_page.to_hash, attrs)
 
+
```
Comment (piotaixr) [inline app/views/layouts/_side.html.haml]: Same, blankline not needed here :)

File: config/aggregators/vat_register.xml [hunk 4]
```diff
@@ -1,44 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (piotaixr) [inline config/aggregators/vat_register.xml]: Should this be removed?

File: db/structure.sql [hunk 5]
```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
```
Comment (piotaixr) [inline db/structure.sql]: Should not be committed unless you are adding a migration. Can you remove it from the PR?

File: app/services/pending_vat_printer.rb [hunk 6]
```diff
@@ -129,7 +129,7 @@ def compute_dataset
 
   def run_pdf
     dataset = compute_dataset
-    pp dataset[-2..-1]
+    dataset[-2..-1]
```
Comment (piotaixr) [inline app/services/pending_vat_printer.rb]: You also probably not need this ;-)

File: app/views/backend/tax_declarations/index.html.haml (hunk 2 above)
Comment (Aquaj) [reply app/views/backend/tax_declarations/index.html.haml]: I think the line adds to readability here.

File: app/views/layouts/_side.html.haml (hunk 3 above)
Comment (Aquaj) [reply app/views/layouts/_side.html.haml]: This one isn't needed though.

**Pull request merged**